src/Application/Actions/Donations/Create.php

```diff
@@ -33,7 +36,8 @@
 
 class Create extends Action
 {
-    private bool $bypassStripe = false;
+    private const MAX_CREATE_RETRY_COUNT = 4;
+    private bool $bypassStripe;
```

Comment:
having bypassStripe here seems OK as a temp thing for CC23 but imho we wouldn't want to keep it permanently as it's putting too much complexity into this class - instead we should depend on something that may or may not bypass stripe. Would you agree @NoelLH ?

Reply:
Right I misread this, thought it was a new line here.